ArjunaJTA/jakartaee_transactional_app/README.md

```diff
@@ -9,11 +9,10 @@ bean and then test it using testsuite deployed in WildFly using Arquillian
 
 This command will build and test the application using Arquillian
 
-Get a WildFly that has the expected version of Narayana in it. If you need to build it:
+Get a WildFly that has the expected version of Narayana in it. If you need to obtain it:
 ```
-WORKSPACE=<QUICKSTARTS_REPO_ROOT> <QUICKSTARTS_REPO_ROOT>/scripts/hudson/quickstart.sh clone_as
-WORKSPACE=<QUICKSTARTS_REPO_ROOT> <QUICKSTARTS_REPO_ROOT>/scripts/hudson/quickstart.sh build_as
-#You should be able to find a WildFly to use for JBOSS_HOME under <QUICKSTARTS_REPO_ROOT>jboss-as/build/target/wildfly-<BUILT_WILDFLY_VERSION>
+WORKSPACE=<QUICKSTARTS_REPO_ROOT> <QUICKSTARTS_REPO_ROOT>/scripts/hudson/quickstart.sh download_and_update_as
+#You should be able to find a WildFly to use for JBOSS_HOME under <QUICKSTARTS_REPO_ROOT>/wildfly-<WILDFLY_VERSION>
```

Comment:
It doesn't seem to change it to a heading (looking at https://github.com/tomjenkinson/quickstart/tree/JBTM-3844-download_and_update/ArjunaJTA/jakartaee_transactional_app). It's meant to be like a comment on bash. But I can move it into the text. It will be better that way I think,